src/main/java/chess/ChessBoard.java

```diff
@@ -20,6 +20,18 @@ public ChessBoard() {
         board = new  Piece[CHESSBOARD_SIZE][CHESSBOARD_SIZE];
         createPieces();
     }
+    public ChessBoard(final Piece piece) {
+        board = new  Piece[CHESSBOARD_SIZE][CHESSBOARD_SIZE];
+        board[piece.getPosition().getColumn()][piece.getPosition().getRow()] = piece;
+    }
+    /**create the cheassboard without piecese**/
+    public void crateChessBoard() {
```

Comment:
Maybe, a more descriptive name should be "clearChessBoard".

Reply:
Change made can you review it?